### ISSUE TITLE
Why imagine - why not actually use a screen reader

### ISSUE THREAD

**Issue by computamike** [MicrosoftDocs/microsoft-style-guide#57]
**Keep paragraphs short and sentence structure simple—aim for one verb per sentence. Read text aloud and imagine it spoken by a screen reader.**

Perhaps we should be using screen readers to 'proofread' our sentences to make sure that documentation is accessible to all - Are there any guides to using a screen reader to read text?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8371785f-17a3-906b-ba39-0efd63e8f196
* Version Independent ID: 27f09f69-0943-156c-2227-e936e91c1348
* Content: [Writing for all abilities - Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/accessibility/writing-all-abilities#feedback)
* Content Source: [styleguide/accessibility/writing-all-abilities.md](https://github.com/MicrosoftDocs/microsoft-style-guide/blob/master/styleguide/accessibility/writing-all-abilities.md)
* Product: **non-product-specific**
* GitHub Login: @pallep
* Microsoft Alias: **pallep**
